httpcore/_sync/http11.py

```diff
@@ -191,8 +180,11 @@ def _receive_event(self, timeout: Optional[float] = None) -> H11Event:
                     raise RemoteProtocolError(msg)
 
                 self._h11_state.receive_data(data)
+            elif event is h11.PAUSED:
+                # TODO: Implement handling for paused
```

Comment:
paused is handled here https://github.com/encode/httpcore/pull/526/files#diff-7b355550eef6173e16a39fc2fd633b09e9589e707ed9d2792dbb3e205cbd7f0eR155